yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiProperties.java

```diff
@@ -69,9 +71,9 @@ public boolean equals(final Object o) {
 
         final YotiProperties that = (YotiProperties) o;
 
-        if (applicationId != null ? !applicationId.equals(that.applicationId) : that.applicationId != null)
+        if (!Objects.equals(applicationId, that.applicationId))
             return false;
-        return scenarioId != null ? scenarioId.equals(that.scenarioId) : that.scenarioId == null;
+        return Objects.equals(scenarioId, that.scenarioId);
```

Comment:
In the same spirit as `ClientProperties`
```
return Objects.equals(applicationId, that.applicationId)
         && Objects.equals(scenarioId, that.scenarioId);
```

Reply:
dropped it